addons/portal_rating/static/src/xml/portal_chatter.xml

```diff
@@ -43,6 +48,15 @@
                 <t t-call="portal.chatter_message_count"/>
             </t>
         </t>
+        <t t-jquery="t[t-call='portal.chatter_messages']" t-operation="before">
+            <div class="o_portal_chatter_messages_loading d-none">
```

Comment:
Wouldn't adding `min-vh-40` here also limit footer "jumps" ?

Reply:
Actually, it doesn't work because min-vh-40 is not defined. Only min-vh-100 is. I have done that styling in css then and tested. Thanks